README.md

```diff
@@ -1,54 +1,190 @@
 # Sudo Sandwich is an example Heroku Add-on
```

Comment:
is there a way to have github create a table of contents for us, or would it make sense to manually make one ourselves?